test/distributed/org/apache/cassandra/distributed/test/sai/IndexAvailabilityTest.java

```diff
@@ -169,6 +170,90 @@ public void testSkipNonQueryableNodeN1Rf1() throws Exception
         shouldSkipNonQueryableNode(1, Collections.singletonList(1));
     }
 
+    @Test
+    public void testIndexExceptionsTwoIndexesOn3NodeCluster() throws Exception
```

Comment:
I needed an explicit test for what we are working on here, but we may just integrate some of the conditions in the other existing tests. This is just a draft